### ISSUE TITLE
Home page navbar not responding correctly at tablet width

### ISSUE THREAD

**Issue by willingc** [napari/napari-sphinx-theme#178]
### 🐛 Bug Report

Looking at the website’s home page on an IPad vertical orientation shoves the top navbar to the right side and overwrites other content

### 💡 Steps to Reproduce

1. Narrow screen width until you see the behavior.

All other pages appear to render correctly.

### 💡 Expected Behavior

clean navbar placement

### 🌎 Environment

ipad

**Comment by willingc** [comment]
@melissawm I think that part of the problem is in the underlying pydata sphinx theme.

<img width="1103" alt="Screenshot 2024-11-13 at 9 55 19 AM" src="https://github.com/user-attachments/assets/5374b096-1e62-4ba1-a36e-813d13ef76c3">
<img width="1051" alt="Screenshot 2024-11-13 at 9 55 51 AM" src="https://github.com/user-attachments/assets/e4b02046-aea1-41b7-9fc7-0fac1de0606e">

**Comment by melissawm** [comment]
Yup that looks like it may be pst - I might be able to take a look there actually. 

We recently just added the community calendar to the landing page so I'm not sure folks would want to walk that back but I can come up with some mockups at least and see if it helps.

**Comment by willingc** [comment]
Thanks. It's somewhat unfortunate as is since an IPad held vertically falls right in the width that looks garbled.
